### PR TITLE
Candidate card expansion

### PR DESCRIPTION
**Pull request by gSoD17**

### CODE REVIEW

**Comment by domdreams** [comment]
Hey @gSoD17  these look amazing! I just have a couple changes. So sorry if they seem nit picky just want to tweak some minor details.

- Can we have all the data labels be bold, 16px and to color $vvgrey (i need to change this to #4e4e4e)
- Is it possible to have commas in numbers over 1,000?
- Can you reduce the size of the helper tip icon to 10px and change the color to $vvgrey
- Please revert back to the colors in the chart and remove the icons. The icons are nice but we want this section to be flexible for different industries. Here are the colors if you need them (#007431, #00903D, #00AF4A, #00D359, #00FC6A)
- Left align the cell content in the industry table, change the text in it to be $vvgrey, 13px expect for the percents which will be a larger 20px and bold
- Can you make sure the full details button includes the button styles mixin (there should be a box shadow) 
- Add padding between the raised/donors and donor/avg donation. 5 pixels or so they are overlapping on some screen sizes

Let me know if you have any questions or if you want me to jump in and help with these changes. Definitely don't want to but you with a bunch of minor design tweaks.

File: src/app/app.module.ts [hunk 1]
```diff
@@ -6,12 +6,16 @@ import { AppComponent } from './app.component';
...
+  MatExpansionModule,
   MatIconModule,
   MatListModule,
   MatSidenavModule,
+  MatTableModule,
   MatToolbarModule,
+  MatTooltipModule,
 } from '@angular/material';
-import { MatExpansionModule } from '@angular/material/expansion';
+import { ChartsModule } from 'ng2-charts';
```
Comment (nancyheiss) [inline src/app/app.module.ts:18]: Looks like we're not following import sorting rules?  https://eslint.org/docs/2.0.0/rules/sort-imports

File: src/app/candidate.ts [hunk 2]
```diff
@@ -3,4 +3,4 @@ export interface Candidate {
     title: string;
     raised: string;
     donors: string;
-  }
\ No newline at end of file
+}
```
Comment (nancyheiss) [inline src/app/candidate.ts:6]: Please add a newline at the end.

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+      <canvas baseChart
+        [chartType]="stackedHorizontalBarChartType"
+        [datasets]="stackedHorizontalBarChartData"
+        [colors]="stackedHorizontalBarChartColors"
+        [options]="stackedHorizontalBarChartOptions"
+        [plugins]="chartPlugins"
+      ></canvas>
+    </div>
+  </div>
+</div>
```
Comment (nancyheiss) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.html:200]: Please add a newline here.

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.ts [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+  templateUrl: './candidate-card-expanded.component.html',
+  styleUrls: ['./candidate-card-expanded.component.scss']
+})
+export class CandidateCardExpandedComponent implements OnInit {
+  @Input() candidate: Candidate;
+  @Input() candidateImg: string;
+  @Output() isExpanded = new EventEmitter<boolean>();
+
+  // Industry Table
+  displayedColumns: string[]
```
Comment (nancyheiss) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.ts:28]: Is this missing a semi-colon at the end? (I don't know ts).

File: src/app/components/candidate-card-expanded/candidate-card-expanded.component.ts [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+  setDisplayedColumns() {
+    this.displayedColumns = [
+      'colorCode',
+      'industry',
+      'amount',
+      'percentage',
+    ];
+  }
+
+  kNumberFormatter(num: number) {
```
Comment (nancyheiss) [inline src/app/components/candidate-card-expanded/candidate-card-expanded.component.ts:243]: Some comments to explain what's going on here would be helpful.

File: src/app/components/candidate-card/candidate-card.component.html [hunk 6]
```diff
@@ -20,5 +21,6 @@ <h2>Donors</h2>
       <h3>{{data[0]["raised vs spent"][0].Donors}}</h3>
     </div>
   </div>
-  <a class="cta">See Details</a> 
+
+  <button class="candidate-button p-col-12" mat-flat-button (click)="outputCandidateData()">See Details</button>
 </div>
```
Comment (nancyheiss) [inline src/app/components/candidate-card/candidate-card.component.html:26]: Please add a newline at the end.

File: src/app/components/mayor/mayor.component.scss [hunk 7]
```diff
@@ -1,6 +1,3 @@
 .candidate-cards{
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: left;
     padding:10px 20px;
 }
```
Comment (nancyheiss) [inline src/app/components/mayor/mayor.component.scss:3]: Please add a newline here.

**Pull request merged**